airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -486,11 +486,11 @@ def generate_scd_type_2_model(self, from_table: str, column_names: Dict[str, Tup
     {{ cursor_field }} as _airbyte_start_at,
     lag({{ cursor_field }}) over (
         partition by {{ primary_key }}
-        order by {{ cursor_field }} desc, _airbyte_emitted_at desc
+        order by {{ cursor_field }} desc nulls last, _airbyte_emitted_at desc
```

Comment:
Thanks for the PR, the only issue now is to check that this is valid SQL syntax for all destinations... 

(or to switch it into a boolean as the first "order by" criterion instead)